code/modules/paperwork/filingcabinet.dm

```diff
@@ -99,30 +99,39 @@
  * Security Record Cabinets
  */
 /obj/structure/filingcabinet/security
-	var/virgin = 1
+	var/virgin = TRUE
```

Comment:
this is a bad variable name - I appreciate that it was like this but if we're touching the code can we give it a more useful name?